admin/requirements-dev.txt

```diff
@@ -1,6 +1,4 @@
-flake8
-pytest
-pytest-cov
-pytest-selenium
-WebTest>=1.3.1
+# Used for development: please do not remove this file
+
 waitress
+pydevd
```

Comment:
What's this package (pypi description is missing...)

Reply:
It is a debugger (client part of eclipse pydev debugger, also works with IntelliJ / PyCharm) as this image is only used for development, I think we could install debuggers by default to ease development.

Reply:
And it's working out of the box with pserve?

Reply:
Related card: https://jira.camptocamp.com/browse/GSGMF-848